.github/workflows/roll_torch_mlir.yml

```diff
@@ -0,0 +1,65 @@
+name: Roll PyTorch
```

Comment:
It isn't clear to me from this PR and the code what exactly this is trying to do. Can you add PR description and maybe a comment in the file?

* Is this creating pull requests that trigger CI jobs?
* Is it committing directly to `main`?
* How are commits selected?